clayui.com/src/components/clay/Editor.js

```diff
@@ -109,10 +114,27 @@ const Editor = ({code, disabled = false, imports, preview = true, scope}) => {
 					<LiveError />
 				</div>
 			)}
+
 			<div className="gatsby-highlight">
 				<div style={{padding: '10px'}}>
-					{imports && <LiveEditor disabled value={imports} />}
-					<LiveEditor />
+					<ClayButtonWithIcon
+						className="btn-collapse"
+						displayType="unstyled"
+						onClick={() =>
+							setCollapseCode(collapseCode => !collapseCode)
```

Comment:
Callback functions are great in `set*`, but in this case we can actually just do this

```suggestion
							setCollapseCode(!collapseCode)
```